.github/workflows/ci_test_package.yml

```diff
@@ -20,6 +20,8 @@ env:
   DBT_ENV_SECRET_DATABRICKS_HTTP_PATH: ${{ secrets.DATABRICKS_HTTP_PATH }}
   DBT_ENV_SECRET_DATABRICKS_TOKEN: ${{ secrets.DATABRICKS_TOKEN }}
   DBT_ENV_SECRET_GCP_PROJECT: ${{ secrets.GCP_PROJECT }}
+  # Env var to test version
+  VERSION_FOR_DOUBLE_RUN: 1.5.0
```

Comment:
Do you thinking setting this as a secret would be worthwhile so we won't have to merge a PR to update this every time we do a release?

Also not sure about the naming. Could we not just use `LATEST_RELEASE_VERSION` ?